rsql-jpa/src/main/java/io/github/perplexhub/rsql/SortUtils.java

```diff
@@ -88,12 +88,14 @@ private static Order sortToJpaOrder(final String[] parts, final SortSupport sort
      * @param context  the rsql context
      * @param property the property
      * @param builder  the criteria builder
+     * @param ic a flag indicating if sort is case insensitive
      * @return the jsonb expression
      */
     private static Expression<?> sortExpressionOfJson(RSQLJPAContext context,
                                                       String property,
                                                       Map<String, String> mapping,
-                                                      CriteriaBuilder builder) {
+                                                      CriteriaBuilder builder,
+                                                      boolean ic) {
```

Comment:
we don't need this extra parameter has the Expression will be casted to String later.